ElementX/Sources/Services/Room/RoomProxyProtocol.swift

```diff
@@ -64,6 +64,13 @@ enum KnockRequestsState {
     case loaded([KnockRequestProxyProtocol])
 }
 
+struct RtcDeclinedEvent {
```

Comment:
Following Swift's API design guidelines this should be:
```suggestion
struct RTCDeclinedEvent {
```